tests/baselines/reference/decoratorMetadataWithImportDeclarationNameCollision4.errors.txt

```diff
@@ -1,4 +1,4 @@
-tests/cases/compiler/service.ts(1,8): error TS1192: Module '"tests/cases/compiler/db"' has no default export.
+tests/cases/compiler/service.ts(1,8): error TS6222: Module '"tests/cases/compiler/db"' has no default export. Did you mean to use 'import {db} from "tests/cases/compiler/db"' instead?

```

Comment:
I think it's more idiomatic to see a space around the identifier in the curlies: `import { db } from ...`.
Basing this on the formatting for destructuring assignment in the TS codebase, as well as how prettier formats imports, to the best of my knowledge.